deps/imgui/CMakeLists.txt

```diff
@@ -71,5 +73,6 @@ elseif("${POLYSCOPE_BACKEND_OPENGL_MOCK}")
 endif()
 
 
+target_compile_features(imgui PUBLIC cxx_std_17)
```

Comment:
Let's definitely remove this, we certainly don't want to increase the C++ version requirement on Polyscope 😄 

(I couldn't find anything about a C++ version bump in imgui, I somehow doubt those folks would jump on C++17)

Reply:
You're right that we can bump it down to C++11. However, if don't set anything it tries to compile in C++98 mode which of course didn't work on my machine.